src/components/HearingCard.js

```diff
@@ -0,0 +1,54 @@
+import React, {PropTypes} from 'react';
+import {FormattedMessage} from 'react-intl';
+import {Link} from 'react-router';
+import formatRelativeTime from '../utils/formatRelativeTime';
+import Icon from '../utils/Icon';
+import LabelList from './LabelList';
+import getAttr from '../utils/getAttr';
+import {getHearingURL} from '../utils/hearing';
+
+const HearingCard = ({hearing, language}) => {
+  const imageStyle = {
+    backgroundImage: hearing.main_image && hearing.main_image.url ? `url(${hearing.main_image.url})` : ''
+  };
+  // FIXME: Should there be direct linking to hearing using certain language?
+  const translationAvailable = !!getAttr(hearing.title, language/* , {exact: true} */);
+
+  return (
+    <div className="hearing-card">
```

Comment:
Actually, link to the hearing when clicking anywhere in the card area. The image is a bigger target to hit than just the title.